plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployJob.java

```diff
@@ -112,12 +113,16 @@ public IStatus runInWorkspace(IProgressMonitor monitor) throws CoreException {
           return cloudSdkProcessStatus;
         }
         // temporary way of error handling, after #439 is fixed, it'll be cleaner
-        return StatusUtil.error(getClass(), "Staging failed, check the error message in the Console View");
+        String errorMessage =
+            getErrorMessageOrDefault(Messages.getString("deploy.job.staging.failed"));
+        return StatusUtil.error(getClass(), errorMessage);
```

Comment:
Yeah, the problem is that the error message is not prefixed by the 'ERROR` string. I wonder if we should file bug against CloudSDK to standardize their error message format versus implementing a pattern matching against all formats we encounter. I guess it should be the former.